processor/addattributesprocessor/testdata/config.yaml

```diff
@@ -0,0 +1,18 @@
+receivers:
+  examplereceiver:
+
+processors:
+  attributes:
+  attributes/2:
+    values:
+      attribute1: 123
+      "string attribute": "string value"
```

Comment:
add some key with '.', e.g.: "http.method", we need to ensure that this doesn't break.

Reply:
Added. Please check again.